src/utils/createArtifactsContent.js

```diff
@@ -269,4 +278,40 @@ const createFeaturesRowData = artifact => {
   }
 }
 
+const getFeatureSetTargetCellValue = targets => {
+  const icons = []
+
+  if (targets) {
+    targets.forEach(target => {
+      if (target.kind === 'nosql') {
+        icons.push({
+          icon: <Nosql />,
+          tooltip: 'NoSql'
+        })
+      } else if (target.kind === 'stream') {
+        icons.push({
+          icon: <Stream />,
+          tooltip: 'Stream'
+        })
+      } else if (target.kind === 'tsdb') {
+        icons.push({
+          icon: <TsdbIcon />,
+          tooltip: 'TSDB'
+        })
+      } else {
+        icons.push({
+          icon: <DbIcon />,
+          tooltip: target.kind
+        })
+      }
+    })
+  }
```

Comment:
Whenever I see this pattern:

```js
 // 1. initializing an identifier to an _empty_ array
const destArray = []

// 2. iterating another array
srcArray.forEach(value => {
  // ...
  // 3. pushing to one array for each item of another array
  destArray.push(/* derived from `value` */)
});

// 4. using the first array. For example, returning it
/* bla bla */ destArray /* bla bla */
```

I think it is a classic case to use `Array#map` method instead:

```js
const destArray = srcArray.map(value => {
    // ..
    return /* derived from value */
})
/* bla bla */ destArray /* bla bla */
```

```js
const kindToIcon = {
  nosql: {
    icon: <Nosql />,
    tooltip: 'NoSql'
  },
  stream: {
    icon: <Stream />
    tooltip: 'Stream'
  },
  tsdb: {
    icon: <TsdbIcon />,
    tooltip: 'TSDB'
  }
}
// option 1:
const icons = (targets ?? []).map(target => ({
  icon: kindToIcon[target.kind]?.icon ?? <DbIcon />,
  tooltip: kindToIcon[target.kind]?.tooltip ?? target.kind
}))
// option 2: (I prefer it)
const icons = (targets ?? []).map(target => kindToIcon[target.kind] ?? {
  icon: <DbIcon />,
  tooltip: target.kind
})
return {
  value: icons,
  class: 'artifacts_small artifacts__targets-icon',
  type: 'icons'
}
```